template/accordion/accordion-group.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel {{panelClass || 'panel-default'}}">
```

Comment:
I think we do want the ng-class to replace the interpolated class, so
```html
<div class="panel" ng-class="panelClass || 'panel-default'">
```
This is to keep behavior consistent with dynamic class changing.